oak-lucene/src/main/java/org/apache/jackrabbit/oak/plugins/index/lucene/util/IndexDefinitionBuilder.java

```diff
@@ -567,21 +576,78 @@ private static Tree getOrCreateChild(Tree tree, String name){
     }
 
     static class SelectiveEqualsDiff extends EqualsDiff {
+
+        List<String> ignorablePropertiesList = ImmutableList.of(FulltextIndexConstants.PROP_NAME,FulltextIndexConstants.PROP_WEIGHT,FIELD_BOOST,
```

Comment:
I'd remove facet related ones from there too (or it'd great if you push that too :)

Reply:
changes done.